openhands/core/config/llm_config.py

```diff
@@ -77,6 +78,7 @@ class LLMConfig:
     log_completions: bool = False
     log_completions_folder: str = os.path.join(LOG_DIR, 'completions')
     draft_editor: Optional['LLMConfig'] = None
+    native_tool_calling: str | bool = 'default'
```

Comment:
we may need to convert strings like "false" to bool, like there is some code in *_config.py doing it for some other bool; the values can come from toml or from env, and from env they're strings

Reply:
Thx for catching that! I actually considered the str-to-bool conversion since I’ve seen similar logic in *_config.py, but I overlooked it this time because it worked fine in the config. Didn’t think about env values being strings and needing explicit conversion—will fix this. Appreciate it!